### PR TITLE
AllUserShellHistory was collecting configs, not shell history.

### PR DESCRIPTION
**Pull request by sebastianwelsh**
[Review on Reviewable](https://reviewable.io/reviews/forensicartifacts/artifacts/119)

### CODE REVIEW

File: definitions/linux.yaml [hunk 1]
```diff
@@ -128,8 +128,8 @@ sources:
 - type: ARTIFACT
   attributes:
     names:
-      - UsersShellConfigs
-      - RootUserShellConfigs
+      - UsersShellHistory
+      - RootUserShellHistory
 labels: [Configuration Files]
```
Comment (joachimmetz) [inline definitions/linux.yaml]: Is this still correct "[Configuration Files]" ? Seeing that these are now history files?

Comment (sebastianwelsh) [inline definitions/linux.yaml]: It's consistent with the UsersShellHistory and RootUserShellHistory artifact labels. Happy to change them, but I'm not sure what category you'd prefer.

Comment (joachimmetz) [inline definitions/linux.yaml]: This is fine for this PR, I've created: https://github.com/ForensicArtifacts/artifacts/issues/123

Can you sync with HEAD and I'll merge.

**Comment by sebastianwelsh** [comment]
OK, sync complete. PTAL.

**Comment by joachimmetz** [comment]
LGTM

**Pull request merged**